beacon-chain/sync/sync_context.go

```diff
@@ -0,0 +1,55 @@
+package sync
```

Comment:
Recommend to renaming this file to `context.go` since the folder name already implies it's going to be sync related